site/sidebar.md

```diff
@@ -90,5 +90,7 @@
 - [Guides](guides/)
   - [The Rationale behind kpt](guides/rationale.md)
   - [Tenant Onboarding](guides/tenant-onboarding.md)
+  - [Porch Installation Guide](guides/porch-installation.md)
+  - [Porch User Guide](guides/porch-user-guide.md)
```

Comment:
You will have to make this change in the template file ---> scripts/generate_site_sidebar/sidebar_template.md.tmpl

and run `make site-generate` that will update the sidebar here.

Reply:
thanks! fixed